src/commands/githubActions/bumpUpstream/index.ts

```diff
@@ -28,11 +29,13 @@ export const gaBumpUpstream: CommandModule<
 export async function gaBumpUpstreamHandler({
   dir = defaultDir
 }: CliGlobalOptions): Promise<void> {
-  const { manifest, format } = readManifest({ dir });
-  const compose = readCompose({ dir });
+  const manifest = readReleaseFile(ReleaseFileType.manifest, { dir });
+  const compose = readReleaseFile(ReleaseFileType.compose, { dir });
 
-  const upstreamRepos = parseCsv(manifest.upstreamRepo);
-  const upstreamArgs = parseCsv(manifest.upstreamArg || "UPSTREAM_VERSION");
+  const upstreamRepos = parseCsv(manifest.releaseFile.upstreamRepo);
```

Comment:
It's strange to have to do `manifest.releaseFile.upstreamRepo` instead of `manifest.upstreamRepo`. In reality you have placed the Manifest type in `manifest.releaseFile`, so the variable you call `manifest` is not really the manifest but something else. What positive value is this adding? At least for my subjective opinion it adds confusion and boilerplate

Reply:
changed to 
```
{ data: ... , format: ...}
``` 

I think this is way more accuarate. The structure is needed to be able to abstract the function that reads, writes and validates. I think it worths in order to simplify code